src/io/iter_image_recordio_2.cc

```diff
@@ -379,6 +379,24 @@ inline void ImageRecordIOParser2<DType>::ParseChunk(dmlc::InputSplit::Blob * chu
           - normalize_param_.max_random_illumination) * normalize_param_.scale;
```

Comment:
normalize_param_.scale is multiplied twice?

Reply:
my bad, mislead by the scale when mean is 0, should remove that.